README.md

```diff
@@ -26,6 +26,25 @@ Go to the [Contributing guide](CONTRIBUTING.md) to learn how to get involved.
 ## Prerequisites
 
 - Node.js v14.x
+  - `node -v`
+- NPM 7 (npm 6 ships with node 14)
+  - `npm install --global npm@7`
+  - `npm -v`
```

Comment:
Can we add a note here, something like:

Caution: Before working with other nodejs 14 projects that are using npm 6, run `npm uninstall --global npm` to revert to npm 6 so that you do not modify the lockfile version in those projects.

Reply:
hmmm. thinking on that one

Reply:
Personally I use [asdf](https://asdf-vm.com/) for managing node versions. We could potentially add an `.nvmrc` file, which is compatible with both `nvm` and `asdf`, and point it to nodejs 16. As long as nothing in the build system actually looks at `.nvmrc`.